samples/net/sockets/http_get/Kconfig

```diff
@@ -0,0 +1,22 @@
+# Private config options for http_get sample
+
+# Copyright (c) 2022 Nuvoton Technology Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+if WIFI
+
+config SAMPLE_HTTP_GET_WIFI_SSID
+	string "WIFI SSID - Network name"
+	default "SSID"
+
+config SAMPLE_HTTP_GET_WIFI_PSK
+	string "WIFI PSK - Network password key"
+	default "PASSWORD"
+
+config SAMPLE_HTTP_GET_WIFI_CONNECT_TIMEOUT_MS
+	int "WIFI connect timeout in milliseconds"
+	default 10000
+
+endif
```

Comment:
I am not sure we want to put dedicated wifi support into every sample app. We could add wifi config option support into setting library for example.

Reply:
I agree, I don't think that generic samples should have dedicated code to deal with L2 connection handling (which would be different for Wi-Fi, LTE, OpenThread etc.). We've (Nordic) opened an issue with a proposal on how to tackle the problem (https://github.com/zephyrproject-rtos/zephyr/issues/52695) - open for discussion. This will also be on an agenda for tomorrow's net forum.